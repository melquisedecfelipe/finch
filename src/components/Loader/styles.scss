.loader {
  background: #FFFFFF;
  box-shadow: 0px 10px 20px #16319B29;
  border-radius: 4px;
}
