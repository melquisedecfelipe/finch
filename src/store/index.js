import { createStore } from 'redux';
// eslint-disable-next-line import/no-extraneous-dependencies
import { composeWithDevTools } from 'redux-devtools-extension';

import rootReducer from './ducks';

const store = createStore(rootReducer, composeWithDevTools());

export default store;
