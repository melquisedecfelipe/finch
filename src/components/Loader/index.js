import React from 'react';
import ContentLoader from 'react-content-loader';

import './styles.scss';

export default function Loader() {
  return (
    <ContentLoader
      className="loader"
      height={335}
      width={300}
      speed={3}
      primaryColor="#f5f5f5"
      secondaryColor="#eeeeee"
    >
      <rect x="0" y="0" rx="0" ry="0" width="350" height="165" />

      <rect x="20" y="190" rx="6" ry="6" width="100" height="25" />
      <rect x="200" y="192.5" rx="6" ry="6" width="90" height="15" />

      <rect x="20" y="225" rx="6" ry="6" width="170" height="25" />

      <rect x="20" y="260" rx="6" ry="6" width="250" height="12.5" />
      <rect x="20" y="280" rx="6" ry="6" width="245" height="12.5" />
      <rect x="20" y="300" rx="6" ry="6" width="210" height="12.5" />
    </ContentLoader>
  );
}
