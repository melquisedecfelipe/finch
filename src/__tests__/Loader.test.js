import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import Loader from '../components/Loader';

import store from '../store';

describe('Loader component testing with Enzyme', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <Loader />
      </Provider>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
