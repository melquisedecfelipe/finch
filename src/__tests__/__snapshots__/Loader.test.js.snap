// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component testing with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;
